docs/user/FlowVariables.md

```diff
@@ -0,0 +1,268 @@
+public platforms supported by the OpenROAD flow.
+
+
+Note:
+-   = indicates default definition assigned by the tool
+-   ?= indicates that the variable value may be reassigned
+-   N/A indicates that the variable/files is not supported currently.
+
+
+| **Configuration Variable**           | **sky130hd** | **sky130hs** | **nangate45** | **asap7** |
+|--------------------------------------|--------------|--------------|---------------|-----------|
+| Library Setup                        |              |              |               |           |
+| `PROCESS`                            | =            | =            | =             | =         |
+| `TECH_LEF`                           | =            | =            | =             | =         |
+| `SC_LEF`                             | =            | =            | =             | =         |
+| `LIB_FILES`                          | =            | =            | =             | =         |
+| `GDS_FILES`                          | =            | =            | =             | =         |
+| `DONT_USE_CELLS`                     | =            | =            | =             | =         |
+| Synthesis                            |              |              |               |           |
+| `LATCH_MAP_FILE`                     | =            | =            | =             | =         |
+| `CLKGATE_MAP_FILE`                   | =            | =            | =             | =         |
+| `ADDER_MAP_FILE`                     | ?=           | ?=           | ?=            | ?=        |
+| `BLACKBOX_MAP_TCL`                   | N/A          | N/A          | N/A           | =         |
+| `TIEHI_CELL_AND_PORT`                | =            | =            | =             | =         |
+| `TIELO_CELL_AND_PORT`                | =            | =            | =             | =         |
+| `MIN_BUF_CELL_AND_PORTS`             | =            | =            | =             | =         |
+| `ABC_CLOCK_PERIOD_IN_PS`             | N/A          | =            | ?=            | ?=        |
+| `ABC_DRIVER_CELL`                    | =            | =            | =             | =         |
+| `ABC_LOAD_IN_FF`                     | =            | =            | =             | =         |
+| Floorplan                            |              |              |               |           |
+| `PLACE_SITE`                         | =            | =            | =             | =         |
+| `TAPCELL_TCL`                        | =            | =            | =             | =         |
+| `MACRO_PLACE_HALO`                   | ?=           | ?=           | ?=            | ?=        |
+| `MACRO_PLACE_CHANNEL`                | ?=           | ?=           | ?=            | ?=        |
+| `PDN_CFG`                            | ?=           | ?=           | ?=            | ?=        |
+| `IO_PIN_MARGIN`                      | =            | =            | =             | =         |

```

Comment:
These should be ?=

Reply:
@maliberty 
need to be updated in platform config.mk first. I'll update it to ?= in this document

Reply:
@vijayank88 I don't think this is a good idea. If you do not modify the `config.mk` you are basically adding new documentation that does not match the current state of the master branch.

Reply:
why is ORFS doc even in OR?

Reply:
@jjcherry56 
As ORFS uses OpenROAD app, @vvbandeira suggested to put this environment variables in OpenROAD docs.

Reply:
The environment variables are not used or respected by anything in OR. They are used in make files in ORFS and as such should be documented there.

Reply:
@jjcherry56 we have centralized documentation in this repo (OR + ORFS) as the docs folder is used to generate the https://openroad.readthedocs.io/en/latest/